ui/app/AppLayouts/Communities/panels/SharedAddressesPanel.qml

```diff
@@ -0,0 +1,680 @@
+            }
+        }
+
+        readonly property bool hasPermissions: root.permissionsModel && root.permissionsModel.count
+
+        readonly property var uniquePermissionTokenKeys: PermissionsHelpers.getUniquePermissionTokenKeys(root.permissionsModel)
+        readonly property var uniquePermissionChannels:
+            root.permissionsModel && root.permissionsModel.count ?
+                PermissionsHelpers.getUniquePermissionChannels(root.permissionsModel, [PermissionTypes.Type.Read, PermissionTypes.Type.ViewAndPost])
+              : []
+
+        // models
+        readonly property var filteredWalletsModel: SortFilterProxyModel {
+            sourceModel: root.walletAccountsModel
+            filters: [
+                ValueFilter {
+                    roleName: "walletType"
+                    value: Constants.watchWalletType
+                    inverted: true
+                }
+            ]
+            sorters: [
+                ExpressionSorter {
+                    function isGenerated(modelData) {
+                        return modelData.walletType === Constants.generatedWalletType
+                    }
+
+                    expression: {
+                        return isGenerated(modelLeft)
+                    }
+                },
+                RoleSorter {
+                    roleName: "position"
+                },
+                RoleSorter {
+                    roleName: "name"
+                }
+            ]
+        }
+
+        readonly property var adminPermissionsModel: SortFilterProxyModel {
+            id: adminPermissionsModel
+            sourceModel: root.permissionsModel
+            function filterPredicate(modelData) {
+                return (modelData.permissionType === Constants.permissionType.admin) &&
+                        (modelData.tokenCriteriaMet && !modelData.isPrivate) // admin privs are hidden if criteria not met
+            }
+            filters: [
+                ExpressionFilter {
+                    expression: adminPermissionsModel.filterPredicate(model)
+                }
+            ]
+        }
+        readonly property var joinPermissionsModel: SortFilterProxyModel {
+            id: joinPermissionsModel
+            sourceModel: root.permissionsModel
+            function filterPredicate(modelData) {
+                return (modelData.permissionType === Constants.permissionType.member) &&
+                        (modelData.tokenCriteriaMet || !modelData.isPrivate)
+            }
+            filters: [
+                ExpressionFilter {
+                    expression: joinPermissionsModel.filterPredicate(model)
+                }
+            ]
+        }
+    }
+
+    ButtonGroup {
+        id: airdropGroup
+        exclusive: true
+    }
+
+    ButtonGroup {
+        id: addressesGroup
+        exclusive: false
+    }
+
+    padding: 0
+
+    contentItem: ColumnLayout {
+        spacing: 0
+
+        // addresses
+        StatusListView {
```

Comment:
Yeah, the same prob (qt bug) with any other component embedding a `Flickable` :/ It's broken with touchpad scrolling only, mouse wheel works fine btw

Reply:
It's also stealing the hover. While the addresses have a custom cursor, the tags scrollView is overriding it.

Reply:
WDYT on creating another task to fix this? We could report it with repro steps on SharedAddressesPopup once it's merged. I think it can be fixed by replacing the tags ScrollView with Flickable and disable the flick if the content size is equal or less to the flickable size.